examples/logging/src/main/java/io/opentelemetry/example/logging/LoggingExporterExample.java

```diff
@@ -19,7 +19,7 @@
   public LoggingExporterExample(OpenTelemetry openTelemetry) {
     tracer = openTelemetry.getTracer(INSTRUMENTATION_NAME);
     counter =
-        GlobalMetricsProvider.getMeter(INSTRUMENTATION_NAME)
+        GlobalMeterProvider.getMeter(INSTRUMENTATION_NAME)
```

Comment:
Apart from removing the user info in the example above. 
There appears "Cannot find symbol" in this logging example. I did minor modifications base on my understanding.